tests/testthat/test-profile_emissions.R

```diff
@@ -22,7 +22,63 @@ test_that("characterize columns", {
   expect_snapshot(names(unnest_company(out)))
 })
 
-test_that("the new API is equivalent to the old API except for extra columns", {
+# test_that("the new API is equivalent to the old API except for extra columns", {
+#   local_options(readr.show_col_types = FALSE)
+#
+#   companies <- read_csv(toy_emissions_profile_any_companies())
+#   co2 <- read_csv(toy_emissions_profile_products_ecoinvent())
```

Comment:
@maurolepore I propose we should remove this test from the repo. This test was created when we shifted from old API to new API. As we start to add new functionalities to the new API, it is difficult to maintain this test which always require to add the new functionalities to the old code as well. Do you think we should remove this test?

Reply:
Sounds good. It looks like it's also time to retire the old API, i.e. stop exporting the `prepare_**tr_product()` and `prepare_**tr_company()` functions. Currently they are only hidden from the documentation with `@keywords internal` but they are still accessible in the public API. 

It would be way cleaner to remove the tests and old API in a separate PR and merge it before this one. That way this PR will be easier to work with and the diff will be focused on what's relevant to this PR alone.

Reply:
Old API is removed through this [PR](https://github.com/2DegreesInvesting/tiltIndicatorAfter/pull/157)